f8a_notification/entryscript.py

```diff
@@ -144,25 +146,24 @@ def remove_cve_versions(pkg_data, new_ver_data):
         name = get_value(result, 'pname')
         eco = get_value(result, 'pecosystem')
         ver = get_value(result, 'version')
-        if 'cve_ids' in result:
-            if pkg_data[eco + ":" + name]['latest'] == ver:
-                del pkg_data[eco + ":" + name]['latest']
-            elif pkg_data[eco + ":" + name]['libio'] == ver:
-                del pkg_data[eco + ":" + name]['libio']
-            if 'libio' not in pkg_data[eco + ":" + name] \
-                    and 'latest' not in pkg_data[eco + ":" + name]:
-                del pkg_data[eco + ":" + name]
-        else:
-            del license_lst[:]
-            if 'licenses' in result:
-                for lic in result['licenses']:
-                    license_lst.append(lic)
-                key = eco + ":" + name + ":" + ver
-                new_ver_data[key] = {}
-                new_ver_data[key]['version'] = ver
-                new_ver_data[key]['package'] = eco + ":" + name
-                new_ver_data[key]['license'] = license_lst
-    print("remove_cve_versions() ended")
+        pkg_key = eco + ":" + name
+
+        del license_lst[:]
```

Comment:
again, consistency in variable naming is one of the good etiquette. e.g. `license_list`